### PR TITLE
hotfix/1.5.5

### PR DESCRIPTION
**Pull request by builtbywill**

### CODE REVIEW

**Comment by builtbywill** [comment]
Ready to review!

File: latex2sympy.py [hunk 1]
```diff
@@ -104,16 +104,37 @@ def process_sympy(self):
     def pre_process_latex(self, latex):
         '''
         pre-processing for issues the parser cannot handle
+
+        find any single char sup/sub and wrap them in "{}"
+        e.g. `4^26^2` => `4^{2}6^{2}`
+        e.g. `x_22` => `x_{2}2`
```
Comment (jkdufair) [inline latex2sympy.py:110]: @builtbywill Do we not allow multi-digit subscripts? Otherwise how do we distinguish between `x_{2}2` and `x_{22}`

Comment (builtbywill) [reply latex2sympy.py:110]: in other words, the front-end would have already wrapped the whole thing as `x_{22}` before it hits this point

the only time in the past it was _not_ wrapped was when it was typed in the front-end as a single char, like the examples above

**Comment by jkdufair** [comment]
Given that tests pass, this is a :thumps: for me!

**Comment by jkdufair** [comment]
Really nice fix.

File: latex2sympy.py [hunk 2]
```diff
@@ -104,16 +104,37 @@ def process_sympy(self):
...
-        # find any single char sup/sub and wrap them in "{}"
-        # e.g. `4^26^2` => `4^{2}6^{2}`
-        # e.g. `x_22` => `x_{2}2`
+        # pattern to find variable commands
+        # match groups:
+        # 0: the variable name up to the first closing "}". could be the "}" for the subscript, or the variable itself (see group 3)
+        # 1: the entire subscript (optional)
+        # 2: the opening subscript bracket (optional)
+        # 3: a second closing "}" (optional). could be from a wrapping command. IF group 2 is NOT "None", this is the closing subscript "}".
+        variable_regex = r'\\variable{([^}_]+(_({)?[^}]+)?)}(})?'
```
Comment (tragessere) [inline latex2sympy.py:123]: @builtbywill This looks great. The only comment I have is that this regex could be updated a bit to capture the whole variable name so you don't need the string join below. The regex below will grab the whole variable name and also won't match on a couple invalid variable names. I have a link with some examples too.

https://regex101.com/r/z6bglU/2
`\\variable{([^}_]+?(_({[^}_]+?}|[^}_]))?)}`

Comment (builtbywill) [reply latex2sympy.py]: @tragessere thanks! that looks great, added it in

**Review by tragessere — APPROVED** [review]
@builtbywill Looks great!

**Pull request merged**